lsp/source/lib/typescript-service.mts

```diff
@@ -445,7 +445,11 @@ function TSHost(compilationSettings: CompilerOptions, initialFileNames: string[]
 }
 
 function TSService(projectURL = "./") {
-  const logger = console
+  const logger = {
```

Comment:
If we need to make `vscode-jsonrpc` work properly, we cannot print redundant logs in the output.